readme.md

```diff
@@ -220,6 +220,9 @@ Integration tests are run under the `it` profile with the Failsafe plugin using
 
 ## Changelog
 
+### 3.2.1 (28 April 2022)
+- Fix JENKINS-63031 slashes no longer break links to Bitbucket
```

Comment:
Should that be [JENKINS-67482](https://issues.jenkins.io/browse/JENKINS-67482) rather than [JENKINS-63031](https://issues.jenkins.io/browse/JENKINS-63031)?